### ISSUE TITLE
Build error with numpy 1.19.0

### ISSUE THREAD

**Issue by Ozon2** [stack-of-tasks/eigenpy#202]
Hello, there is an error compiling eigenpy on Arch Linux with numpy 1.19.0.

```
/home/ozon/.cache/yay/eigenpy/src/eigenpy-2.4.0/include/eigenpy/ufunc.hpp:149:5: erreur: pas de correspondance pour convertir la fonction « binary_op_greater_equal » vers le type « PyUFuncGenericFunction » {aka « void (*)(char**, const long int*, const long int*, void*) »}
  149 |     EIGENPY_REGISTER_BINARY_UFUNC(greater_equal,code,Scalar,Scalar,bool);
      |     ^
/home/ozon/.cache/yay/eigenpy/src/eigenpy-2.4.0/include/eigenpy/ufunc.hpp:48:5: note: les candidats sont : « template<class T> void eigenpy::internal::binary_op_greater_equal(char**, npy_intp*, npy_intp*, void*) »
   48 |     EIGENPY_REGISTER_BINARY_OPERATOR(greater_equal,>=)
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/ozon/.cache/yay/eigenpy/src/eigenpy-2.4.0/include/eigenpy/ufunc.hpp:48:5: note:                      « template<class T1, class T2, class R> void eigenpy::internal::binary_op_greater_equal(char**, npy_intp*, npy_intp*, void*) »
   48 |     EIGENPY_REGISTER_BINARY_OPERATOR(greater_equal,>=)
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/ozon/.cache/yay/eigenpy/src/eigenpy-2.4.0/include/eigenpy/ufunc.hpp:150:5: erreur: pas de correspondance pour convertir la fonction « binary_op_less_equal » vers le type « PyUFuncGenericFunction » {aka « void (*)(char**, const long int*, const long int*, void*) »}
  150 |     EIGENPY_REGISTER_BINARY_UFUNC(less_equal,code,Scalar,Scalar,bool);
      |     ^
/home/ozon/.cache/yay/eigenpy/src/eigenpy-2.4.0/include/eigenpy/ufunc.hpp:47:5: note: les candidats sont : « template<class T> void eigenpy::internal::binary_op_less_equal(char**, npy_intp*, npy_intp*, void*) »
   47 |     EIGENPY_REGISTER_BINARY_OPERATOR(less_equal,<=)
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/ozon/.cache/yay/eigenpy/src/eigenpy-2.4.0/include/eigenpy/ufunc.hpp:47:5: note:                      « template<class T1, class T2, class R> void eigenpy::internal::binary_op_less_equal(char**, npy_intp*, npy_intp*, void*) »
   47 |     EIGENPY_REGISTER_BINARY_OPERATOR(less_equal,<=)
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/ozon/.cache/yay/eigenpy/src/eigenpy-2.4.0/include/eigenpy/ufunc.hpp:153:5: erreur: pas de correspondance pour convertir la fonction « unary_op_negative » vers le type « PyUFuncGenericFunction » {aka « void (*)(char**, const long int*, const long int*, void*) »}
  153 |     EIGENPY_REGISTER_UNARY_UFUNC(negative,code,Scalar,Scalar);
      |     ^
/home/ozon/.cache/yay/eigenpy/src/eigenpy-2.4.0/include/eigenpy/ufunc.hpp:73:5: note: les candidats sont : « template<class T> void eigenpy::internal::unary_op_negative(char**, npy_intp*, npy_intp*, void*) »
   73 |     EIGENPY_REGISTER_UNARY_OPERATOR(negative,-)
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/ozon/.cache/yay/eigenpy/src/eigenpy-2.4.0/include/eigenpy/ufunc.hpp:73:5: note:                      « template<class T, class R> void eigenpy::internal::unary_op_negative(char**, npy_intp*, npy_intp*, void*) »
   73 |     EIGENPY_REGISTER_UNARY_OPERATOR(negative,-)
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
make[2]: *** [unittest/CMakeFiles/user_type.dir/build.make:86 : unittest/CMakeFiles/user_type.dir/user_type.cpp.o] Erreur 1
make[2] : on quitte le répertoire « /home/ozon/.cache/yay/eigenpy/src/eigenpy-2.4.0 »
make[1]: *** [CMakeFiles/Makefile2:1442 : unittest/CMakeFiles/user_type.dir/all] Erreur 2
make[1] : on quitte le répertoire « /home/ozon/.cache/yay/eigenpy/src/eigenpy-2.4.0 »
make: *** [Makefile:164 : all] Erreur 2
==> ERREUR : Une erreur s’est produite dans build().
```

This error probably comes from numpy/numpy#15355.
`pip install 'numpy<1.19.0'` fixes the issue.

**Comment by nim65s** [comment]
Thanks for the report ! I think this can be a relatively low priority, as the workaround is trivial, but the fix seems really easy too: https://github.com/tensorflow/tensorflow/pull/40728

**Comment by jcarpent** [comment]
Thanks a lot @Ozon2 for raising this issue. I've proposed a fix here #203. I will proceed to a new release afterwards.

**Comment by jcarpent** [comment]
Solved by #203.
